Source/WebCore/platform/network/curl/CurlSSLHandle.h

```diff
@@ -107,7 +107,7 @@ class CurlSSLHandle {
 
     String m_cipherList;
     String m_signatureAlgorithmsList;
-    String m_curvesList;
+    String m_ecCurves;
```

Comment:
Converting UTF-8 to UTF-16 and Converting back to UTF-8 doesn't seems nice. How about using `CString` and UTF-8?
curl handle also retains the CURLOPT_SSL_EC_CURVES string. I don't think `CurlSSLHandle` also should retain the string. But, this doesn't seems like a simple change. Just an idea.

Reply:
I use `CString` instead of `String`.